src/Padawan/Framework/Domain/Project/Persister.php

```diff
@@ -26,9 +26,10 @@ public function __construct(PathResolver $path, LoopInterface $loop)
     public function save(Project $project)
     {
         $this->checkForPadawanDir($project->getRootFolder());
-        return $this->fs->file(
-            $this->getProjectIndexFilePath($project->getRootFolder())
-        )->putContents($this->serialize($project));
+        return file_put_contents(
```

Comment:
I'm not familiar with php react but there should be a way to keep this operation asynchronous while promising the file shall be written to disk. Any advise?